### DISCUSSION TITLE
Version 0.6.3

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#396] [Announcements]
Good day! Machinaris v0.6.3 is now available. Changes include:

-  [Staicoin](https://github.com/STATION-I/staicoin-blockchain) - cross-farming support for this blockchain fork.
-  Machinaris - fullnode initialization and setup has been simplified and greatly sped up for all forks.
-  Chia - Update to version 1.2.11.  See their [changelog for details](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.11).

<hr /><em>This discussion was created from the release <a href='https://github.com/guydavis/machinaris/releases/tag/0.6.3'>Version 0.6.3</a>.</em>

**Comment by StefVos** [comment]
What a wonderful release. All works great for me. I am using an [8 years old pc](https://www.hpe.com/psnow/doc/c04111672.pdf?jumpid=in_lit-psnow-getpdf&qsVersion=28&ver=28) with 100TB plots on multiple usb 2 disks. Farming Chia, Flax, Hddcoin and Stai no problem. 8 old laptops do the (very slow) plotting. This project makes Chia farming fun.
Thanks 😀
